client/my-sites/patterns/components/pattern-library/index.tsx

```diff
@@ -276,7 +276,11 @@ export const PatternLibrary = ( {
 				{ ! isHomePage && (
 					<PatternLibraryBody className="pattern-library">
 						<div className="pattern-library__header">
-							<h1 className="pattern-library__title">
+							<h1
```

Comment:
NIT

```
{ searchTerm ? (
	<h1 className="pattern-library__title pattern-library__title--search">
		{ translate( '%(count)d pattern', '%(count)d patterns', {
			count: patterns.length,
			args: { count: patterns.length },
		} ) }
	</h1>
) : (
	<h1 className="pattern-library__title">
		{ patternTypeFilter === PatternTypeFilter.PAGES &&
			translate( 'Page Layouts', {
				comment: 'Refers to block patterns that contain entire page layouts',
			} ) }
		{ patternTypeFilter === PatternTypeFilter.REGULAR &&
			translate( 'Patterns', {
				comment: 'Refers to block patterns',
			} ) }
	</h1>
) }
```

Reply:
This doesn't seem to reduce indentation, so I'll leave the current code in place